adapt/README.md

```diff
@@ -0,0 +1,11 @@
+<!--
+SPDX-FileCopyrightText: 2020 Serokell <https://serokell.io/>
+
+SPDX-License-Identifier: MPL-2.0
+-->
+
+Some of the repositories we depend on are structured in various _original_
```

Comment:
Any chance you can be less snarky?

Reply:
I wonder if it could actually be _more_ snarky than that.

Reply:
It could.

I’m not sure what the point y’all are trying to make here though. Are you saying that the way the `haskell.nix` repository is structured makes any sense?